scripts/github_issue_retriever.py

```diff
@@ -33,6 +67,12 @@ def retrieve_issues(
                 owner, repo, state, last_modified.isoformat(), retrieve_events
             )
 
+            if retrieve_private:
+                logger.info(
+                    "Replacing contents of auto closed public issues with private issues content"
+                )
+                data, _, _ = replace_with_private(data)
```

Comment:
If you apply the nit suggested above, this becomes simply `replace_with_private(data)`